app/src/main/res/values/strings.xml

```diff
@@ -15,17 +15,35 @@
   <!--notes-->
   <string name="title_notes">Notes</string>
   <!--report-->
+  <string name="basic_stats">Basic Stats</string>
   <string name="characters">Characters</string>
+  <string name="characters_without_spaces">Characters without Spaces</string>
   <string name="words">Words</string>
   <string name="paragraphs">Paragraphs</string>
   <string name="size">Size</string>
   <string name="bookmark_hint">Enter Report name</string>
 
+  <!--Extra Stats-->
+  <string name="extra_stats">Extra Stats</string>
+  <string name="unique_words">Unique Words</string>
+  <string name="average_word_length">Average Word Length</string>
+  <string name="avg_sentence_length_words">Avg Sentence Length(Words)</string>
+  <string name="avg_sentence_length_characters">Avg Sentence Length(Characters)</string>
```

Comment:
I think this both should have `Average` at the start instead of `Avg` . Because that will reflect in out translations.

Reply:
With average the text will be multi line on smaller devices.

Reply:
That's fine I guess? `Avg` will fail royally in translations specially as those are going to be automatic.